controllers/users.js

```diff
@@ -28,9 +27,9 @@ router.post('/', function(req, res) {
 // Update a user
```

Comment:
Not a criticism, just a curiosity. Both of the update functions don't do any sort of checking of the parameters sent in the request. Does this mean people can just add random fields to our objects? Or will Node not allow that?